### DISCUSSION TITLE
Feature suggestions

### DISCUSSION THREAD

**Discussion by Mikisepic** [StonksCo/beer-stonks#37] [Ideas]
Feel free to suggest features!

**Comment by wolftherai** [comment]
Beer Stock Market: https://www.beerxchange.com/

**Comment by wolftherai** [comment]
FAQ page (drinking tips).
Separate page for people in need of some knowledge, e.g. (How do I not get drunk so quickly?)

**Comment by wolftherai** [comment]
STATS:
https://www.falstaff.com/en/nd/which-countries-drink-the-most-alcohol/%20data%20taken%20from%20:%20https://ourworldindata.org/alcohol-consumption
